<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CyclePrintSetting</class>
 <widget class="QWidget" name="CyclePrintSetting">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>187</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>474</width>
    <height>237</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>打印参数设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButtonRestoreDefaults">
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QPushButton" name="pushButtonOK">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>沉淀时间(s)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxBreathe1">
     <property name="maximum">
      <double>60.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxOverlift1">
     <property name="maximum">
      <double>9.000000000000000</double>
     </property>
     <property name="value">
      <double>4.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxSettle1">
     <property name="maximum">
      <double>60.000000000000000</double>
     </property>
     <property name="value">
      <double>3.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="spinBoxRaiseSpd1">
     <property name="maximum">
      <number>130</number>
     </property>
     <property name="value">
      <number>120</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="toolTip">
      <string>停留时间</string>
     </property>
     <property name="text">
      <string>Residence(s)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spinBoxLowerSpd1">
     <property name="maximum">
      <number>130</number>
     </property>
     <property name="value">
      <number>120</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Raise Speed(r/min)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Lower Speed(r/min)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Extra heiight(mm)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
